requirements.txt

```diff
@@ -1,4 +1,5 @@
 appdirs
 py-radix
 portion
+lzma
```

Comment:
I guess this should be pylzma?

Reply:
> I guess this should be pylzma?
 
I rectified it. The lzma module is part of the standard library ,So dependency and installation not needed.